ensenso_camera/include/ensenso_camera/conversion.h

```diff
@@ -3,11 +3,21 @@
 #include <geometry_msgs/Point32.h>
 
 #include "nxLib.h"
+#include "ros/ros.h"
```

Comment:
Include sieht unnötig aus. Wenn dann sollte der externe include aber auch in spitze Klammern.